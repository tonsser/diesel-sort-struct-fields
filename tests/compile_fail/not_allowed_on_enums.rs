use diesel_sort_struct_fields::sort_fields;

#[sort_fields]
pub enum Thing {
    A,
    B,
}

fn main() {}
