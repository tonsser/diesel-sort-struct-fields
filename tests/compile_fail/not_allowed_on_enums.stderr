error: `#[sort_fields]` is not allowed on enums, only structs
 --> $DIR/not_allowed_on_enums.rs:4:5
  |
4 | pub enum Thing {
  |     ^^^^
